fuel/src/main/kotlin/com/github/kittinunf/fuel/Fuel.kt

```diff
@@ -4,9 +4,10 @@ import com.github.kittinunf.fuel.core.FuelManager
 import com.github.kittinunf.fuel.core.Method
 import com.github.kittinunf.fuel.core.Parameters
 import com.github.kittinunf.fuel.core.Request
-import com.github.kittinunf.fuel.util.TestConfiguration
+import com.github.kittinunf.fuel.core.requests.DownloadRequest
+import com.github.kittinunf.fuel.core.requests.UploadRequest
 
-class Fuel {
+class Fuel private constructor() {
```

Comment:
I feel like we might take this chance to make this `object`?  `Fuel` is basically a "namespace"

Reply:
Yes!

Reply:
Done, and then some. Hope you like 💯